packages/expo-cellular/src/Cellular.ts

```diff
@@ -13,8 +13,8 @@ export { CellularGeneration };
 // @needsAudit
 /**
  * Indicates if the carrier allows making VoIP calls on its network. On Android, this checks whether
- * the system supports SIP-based VoIP API. See [here](https://developer.android.com/reference/android/net/sip/SipManager.html#isVoipSupported(android.content.Context))
- * to view more information.
+ * the system supports SIP-based VoIP API. See [Android documentation](https://developer.android.com/reference/android/net/sip/SipManager.html#isVoipSupported(android.content.Context))
```

Comment:
```suggestion
 * the system supports SIP-based VoIP API. See the [Android documentation](https://developer.android.com/reference/android/net/sip/SipManager.html#isVoipSupported(android.content.Context))
```

Reply:
Thanks for catching that. Updated it:

![CleanShot 2024-06-05 at 13 32 59@2x](https://github.com/expo/expo/assets/10234615/81e8fd40-83fe-4fae-8b17-67230a62d591)